src/coreclr/src/ildasm/dasm.cpp

```diff
@@ -3475,7 +3475,8 @@ BOOL DumpMethod(mdToken FuncToken, const char *pszClassName, DWORD dwEntryPointT
     bool bRet = FALSE;
 
     PAL_CPP_TRY {
-        if((*pComSig & IMAGE_CEE_CS_CALLCONV_MASK) > IMAGE_CEE_CS_CALLCONV_VARARG)
+        if((*pComSig & IMAGE_CEE_CS_CALLCONV_MASK) > IMAGE_CEE_CS_CALLCONV_VARARG
```

Comment:
I would delete this error handling completely; and instead changed PrettyPrintSig to print `callconv(value)` for values that do not translated to keywords.